ansible/roles/join-ad-linux/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+service_account_ssm_passwords:
+  service_account:
+    parameter: "/join_domain_linux_service_account/passwords"
+    users:
+      - username: auto
```

Comment:
If you set auto, ansible will auto generate a password for you.  You probably don't want this.  Just set blank, e.g.
- username:
- password:
Then manually update the SSM param via console/CLI and set to json "{"username": "myusername", "password": "mypassword"} before running the ansible